templates/base.html

```diff
@@ -148,6 +148,12 @@
 
 						{# Separate menu optimized for mobile users #}
 						<ul class="nav navbar-nav visible-xs">
+							{% notification_count as n_count %}
+								{% if n_count > 0 %}
```

Comment:
The if tag should be indented one level less because the notification_count tag does not start a new block. The if should start in the same level as the notification_count.